tests/playwright/src/model/pages/main-page.ts

```diff
@@ -94,4 +99,33 @@ export abstract class MainPage extends BasePage {
     }
     return undefined;
   }
+
+  async getRowsFromTableByStatus(status: string): Promise<Array<Locator>> {
+    await waitUntil(async () => await this.rowsAreVisible(), { sendError: false });
+
+    const table = this.content.getByRole('table');
+    if (table) {
+      const rows = await table.getByRole('row').all();
+      rows.shift(); //first row is the headers row
+      const filteredRows = [];
+      for (const row of rows) {
```

Comment:
Are you sure that for here does not iterate over the header row also? It seems to me that a classic for with a standard iterator would be better, since you can initialize the iterator from 1, instead of 0.